scripts/upgrade_admin_server.sh

```diff
@@ -100,6 +100,20 @@ n.save"
 
     # Upgrade the distribution non-interactively
     zypper --no-color --releasever 12.3 ref -f
+
+    systemctl stop rabbitmq-server
+    systemctl stop couchdb
+    sleep 5
+    systemctl stop epmd.socket
+    systemctl stop epmd.service
+    sleep 5
+    killall epmd
```

Comment:
What if epmd doesn't react to SIGTERM? shall we wait and try with `killall -9 epmd`? This seems to be the way systemd does stopping by default.

Reply:
also, shouldn't we check for epmd state before killing it?

Reply:
killall is happy to not kill anything :o)